contracts/lido-satellite/src/state.rs

```diff
@@ -0,0 +1,20 @@
+use cosmwasm_schema::cw_serde;
+use cw_storage_plus::Item;
+
+#[cw_serde]
+pub struct Config {
+    pub bridged_denom: String,
+    pub canonical_subdenom: String,
+}
+
+impl Config {
+    pub fn get_full_tokenfactory_denom(&self, contract_address: impl AsRef<str>) -> String {
+        format!(
+            "factory/{}/{}",
```

Comment:
please use the standard query: https://github.com/neutron-org/neutron-sdk/blob/main/packages/neutron-sdk/src/query/token_factory.rs#L18

Reply:
This request has been implemented in #2